naming/src/main/java/com/alibaba/nacos/naming/core/v2/service/impl/PersistentClientOperationServiceImpl.java

```diff
@@ -180,6 +208,12 @@ public Response onApply(WriteRequest request) {
         }
     }
     
+    private void onInstanceChange(Service service, Instance instance, String clientId) {
+        if (clientManager.contains(clientId)) {
```

Comment:
Whether this judgment is accurate？

Reply:
The solution here is that if the instance is logged out, the updated instance will not re register the instance. The logged out "apply" will remove the client, and raft can ensure the order of the logged out "apply" and the updated "apply"

> Whether this judgment is accurate？

Reply:
How about one client register two persistent instance, but only remove one instance?

Reply:
i will fix it